src/main/java/com/example/springresttask/domain/dto/EmailDto.java

```diff
@@ -0,0 +1,25 @@
+package com.example.springresttask.domain.dto;
+
+import javax.persistence.Column;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class EmailDto {
+    @Size(min = 2, max = 100, message = "Name should be 2-100 characters long")
+    @Column(name = "recipient_name")
```

Comment:
DTOs are not persisted, they are data **transfer** objects, therefore they are only for sending data back and forth. This annotation pertains to JPA, and therefore should be removed from here